RELEASE.md

```diff
@@ -0,0 +1,73 @@
+Release type: minor
+
+Update execution to allow adding instances of `Exception` type to `info.context.partial_errors` to get added to the result after execution, allowing users to add errors to the errors array from a field resolver while still resolving that field.
+
+**Usage examples:**
+
+Basic usage:
+
+```python
+import strawberry
+
+
+@strawberry.field
+def query(self, info) -> bool:
+    info.context.partial_errors.append(Exception("Partial failure"))
```

Comment:
I think moving this off of context and into `info` would be preferred, since context can be anything by implementing frameworks/clients.

I'll look into it

Reply:
I've gone with an iffy route of trying to support attribute setting and indexing generically, though it's kind of gross.

Since `info` comes from the graphql core library, I'm not so sure moving it into `info` makes much sense.

Since I don't love either approach, it kind of implies that maybe this isn't a great idea 😅